examples/eyalroz_printf/config.cmake

```diff
@@ -0,0 +1,6 @@
+hunter_config(
+    eyalroz_printf 
+    VERSION 6.2.0-b1 
+    CMAKE_ARGS BUILD_STATIC_LIBRARY=ON
```

Comment:
This is not required. By default, a dynamic version of library is built. I just wanted to see in an example how cmake args are passed.
🤔 Now I think it would be more correct to document the supported args in the `eyalroz_printf.rst`. And with such documentation `config.cmake` will not be needed.
I'll add argument information to .rst and then remove `config.cmake`. What do you say?

Reply:
The standard CMake way would be to use the variable `BUILD_SHARED_LIBS` as shown in the cmake documentation: https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html#variable:BUILD_SHARED_LIBS.

Maybe you could propose this to the library authors as PR, or maybe they already support it. Should be worth a try.

There is even a hunter specific user variable: https://hunter.readthedocs.io/en/latest/reference/user-variables.html#hunter-build-shared-libs

---

If `BUILD_SHARED_LIBS` is not supported removing the `config.cmake` and mentioning it in the `*.rst` documentation is a good idea :+1:

Reply:
@NeroBurner : FYI, I now use BUILD_SHARED_LIBS.